src/main/java/com/github/fabienrenaud/jjb/stream/Deserialization.java

```diff
@@ -106,4 +125,244 @@ public Object mjson() throws Exception {
     public Object underscore_java() throws Exception {
         return JSON_SOURCE().streamDeserializer().underscore_java(JSON_SOURCE().nextString());
     }
+    
+    @Benchmark
+    @Override
+    public Object purejson() throws Exception {
+        final AtomicReference<Object> ref = new AtomicReference<>();
+        Parser.create().parse(JSON_SOURCE().nextString()).ifSuccess(v -> ref.set(PureJson.toObject(v))).ifFailure(System.out::println);
```

Comment:
the test needs to crash in case of failure, not just print. otherwise failure give the best perf.